gossip/src/cluster_info.rs

```diff
@@ -1016,15 +1017,35 @@ impl ClusterInfo {
                 }
             }
         };
+        let (_, vote_tx, hash) = parse_vote_transaction(&vote).unwrap();
```

Comment:
This incurs an extra deserialization just to get the last voted slot. We could just pass that slot into this function